plugin/h5pimport/src/ZipPackageImporter.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+// For licensing terms, see /license.txt
+
+namespace Chamilo\PluginBundle\H5pImport\H5pImporter;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Class ZipPackageImporter.
+ */
+class ZipPackageImporter extends H5pPackageImporter
+{
+    private const ALLOWED_FILES = [
+        'json',
+        'png',
+        'jpg',
+        'jpeg',
+        'gif',
+        'bmp',
+        'tif',
+        'tiff',
+        'svg',
+        'eot',
+        'ttf',
+        'woff',
+        'woff2',
+        'otf',
+        'webm',
+        'mp4',
+        'ogg',
+        'mp3',
+        'm4a',
+        'wav',
+        'txt',
+        'pdf',
+        'rtf',
+        'doc',
+        'docx',
+        'xls',
+        'xlsx',
+        'ppt',
+        'pptx',
+        'odt',
+        'ods',
+        'odp',
+        'xml',
+        'csv',
+        'diff',
+        'patch',
+        'swf',
+        'md',
+        'textile',
+        'vtt',
+        'webvtt',
+        'gltf',
+        'gl',
+        'js',
+        'css',
+    ];
+
+    public function import(): string
```

Comment:
Would be nice to have a PHPDoc block saying something like "Validates then unzip the H5P package into the course folder. No DB change.". Just so we don't have to read the method's code to see what it does.